bundles/org.openhab.binding.wemo/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -176,7 +175,92 @@
 				<label>Unique Device Name</label>
 				<description>The UDN identifies the WeMo Device</description>
 			</parameter>
+		</config-description>
+	</thing-type>
 
+	<thing-type id="Crockpot">
```

Comment:
Curious - where do these thing definitions come from? How come they were missing? Perhaps this should be mentioned in PR description as it seems like something was really broken?

Reply:
When I added the handlers classes in openHAB 2.5, I forgot to checkin the updated thing-types.xml file.
Will add a note in PR description.